galasa-ui/src/components/HomeContent.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+'use client'
+import Image from "next/image";
+import homeGraphic from "../assets/images/homeGraphic.png"
+import { Heading, Section } from "@carbon/react"
+import "../styles/global.scss"
+import styles from "../styles/HomeContent.module.css"
+
+export default function HomeContent() {
+
+    return (
+        <Section level={1}>
+            <div className={styles.homeContentWrapper}>
+                <Heading>Welcome to your Galasa Service</Heading>
+                <Section level={2}>
+                    <h4>Get the most from your Galasa experience by reading the Galasa documentation &nbsp;
```

Comment:
This is being rendered with extra spacing between `documentation` and the `here` link - wouldn't it be better to put the link around `Galasa documentation`?